make_level.py

```diff
@@ -141,38 +116,49 @@ def main():
         "level_name: {}".format(level_name),
         "Confirm[y/n] "
     ]), end='')
-
-    choice = input().lower()
-    if choice == 'n':
-        return
+    
+    choice = ''
+    while choice != 'y':
+        choice = input().lower()
+        if choice == 'n':
+            return
+        elif choice != 'y':
+            print("Confirm[y/n] ", end='')
 
     # Register package
     # Read setup.py
     with open("setup.py", 'r') as fp:
         filedata = fp.read()
 
-    # Update desired lines
+    # Check if skill is already registered.
+    skill_exists = False
+    if "gitgud.skills.{}".format(skill_name) in filedata:
+        skill_exists = True
+
+    # Concatenate potential entry
     replace1 = "\n".join([
         "        \'gitgud.skills.{}\',".format(skill_name),
         "    ],",
         "    package_data"
     ])
 
     replace2 = "\n".join([
-        "        \'gitgud.skills.{}\'['_*/*'],".format(skill_name),
+        "        \'gitgud.skills.{}\': ['_*/*'],".format(skill_name),
         "    },",
         "    python_requires"
     ])
 
-    filedata = filedata.replace("\n".join([
-        "    ],",
-        "    package_data"
-    ]), replace1)
+    # Register new skill to setup.py
+    if not skill_exists:
```

Comment:
It seems like we don't need this if statement or the `skill_exists` variable, we can just use the if statement from before

Reply:
removed